src/sema/assembly/block.rs

```diff
@@ -17,7 +17,7 @@ pub struct AssemblyBlock {
 /// Returns the resolved block and a boolean that tells us if the next statement is reachable.
```

Comment:
The module is still called `assembly` (the one in sema)

Reply:
>  Changes to the sema will be made in another PR to avoid development conflicts.

@seanyoung

Reply:
Ah. Why multiple PRs? Because you're still working on sema in other PRs?

Ideally PRs should be self-contained, complete pieces of work.

Reply:
Ok, I'll change that after merging the other PRs.

Reply:
Ready for review now!

Reply:
Ready for review now!